--- po/Makefile.in.in	2020-04-06 19:12:50.652868150 +0200
+++ po/Makefile.in.in.new	2020-04-06 19:16:31.289415435 +0200
@@ -163,6 +163,11 @@
 # otherwise packages like GCC can not be built if only parts of the source
 # have been downloaded.
 
+# XXX: hack: since we want to parse generated files with xgettext,
+# do not complain if these files are not found in the source tree,
+# because they will be found in the build tree
+$(POTFILES):
+
 # This target rebuilds $(DOMAIN).pot; it is an expensive operation.
 # Note that $(DOMAIN).pot is not touched if it doesn't need to be changed.
 # The determination of whether the package xyz is a GNU one is based on the
@@ -194,7 +199,7 @@
 	fi; \
 	case `$(XGETTEXT) --version | sed 1q | sed -e 's,^[^0-9]*,,'` in \
 	  '' | 0.[0-9] | 0.[0-9].* | 0.1[0-5] | 0.1[0-5].* | 0.16 | 0.16.[0-1]*) \
-	    $(XGETTEXT) --default-domain=$(DOMAIN) --directory=$(top_srcdir) \
+	    $(XGETTEXT) --default-domain=$(DOMAIN) --directory=$(top_builddir) --directory=$(top_srcdir) \
 	      --add-comments=TRANSLATORS: \
 	      --files-from=$(srcdir)/POTFILES.in \
 	      --copyright-holder='$(COPYRIGHT_HOLDER)' \
@@ -202,7 +207,7 @@
 	      $(XGETTEXT_OPTIONS) @XGETTEXT_EXTRA_OPTIONS@ \
 	    ;; \
 	  *) \
-	    $(XGETTEXT) --default-domain=$(DOMAIN) --directory=$(top_srcdir) \
+	    $(XGETTEXT) --default-domain=$(DOMAIN) --directory=$(top_builddir) --directory=$(top_srcdir) \
 	      --add-comments=TRANSLATORS: \
 	      --files-from=$(srcdir)/POTFILES.in \
 	      --copyright-holder='$(COPYRIGHT_HOLDER)' \
